di_website/templates/includes/heroes/hero_home.html

```diff
@@ -0,0 +1,49 @@
+{% load wagtailimages_tags wagtailcore_tags responsive %}
+
+
+{% if hero_image %}
+  {% responsiveimage hero_image fill-1600x780-c100 srcset=global.hero_srcs as image %}
+{% else %}
+  {% responsiveimage page.hero_image fill-1600x780-c100 srcset=global.hero_srcs as image %}
+{% endif %}
+<section class="hero{% if image %} hero--image{% endif %} {% block hero_class %}{% endblock %}">
+  {% if image %}
+      <div class="hero__image" style="background-image: url('{{ image.url }}');"></div>
+      {% if page.hero_image_credit_name %}
+          <span class="hero__credit">Image by
+          {% if page.hero_image_credit_url %}
+              <a href="{{page.hero_image_credit_url}}" target="_blank">{{page.hero_image_credit_name}}</a>
+          {% else %}
+              {{page.hero_image_credit_name}}
+          {% endif %}
+          </span>
+      {% endif%}
+  {% endif %}
+
+  <span class="hero__flourish"></span>
+  <span class="hero__flourish"></span>
+  <span class="hero__flourish"></span>
+  <span class="hero__flourish"></span>
+  <span class="hero__flourish"></span>
+  <span class="hero__flourish"></span>
+  <span class="hero__flourish"></span>
+  <span class="hero__flourish"></span>
+  <div class="row row--narrow">
+      <div class="hero__content">
+        <ul class="hero__meta">
+            <li class="hero__meta-item hero__meta-item--block">{{ page.publication_type|default_if_none:"Publication" }}</li>
+        </ul>
+        <h1 class="hero__title">
+          <span>{{ page.title }}</span>
+        </h1>
+        {% if page.hero_text %}
+          <div class="hero__excerpt"><span>{{ page.hero_text|richtext }}</span></div>
+        {% endif %}
+        {% if page.hero_link and page.hero_link_caption %}
+          <a href="{% pageurl page.hero_link %}" class="button button--reverse">{{ page.hero_link_caption }}</a>
```

Comment:
In this case, it wouldn't make sense for the Home to inherit the `page.hero_link` I think this needs to be refactored to remove the boolean, and always link to `{% pageurl page %}`  with the text `View full report` so it always has a link to the publication page.

Reply:
Ah, true... nice catch.